docs/support/index.rst

```diff
@@ -113,6 +113,19 @@ NAPALM supports passing certain optional arguments to some drivers. To do that y
     >>> device = driver('192.168.76.10', 'dbarroso', 'this_is_not_a_secure_password', optional_args=optional_args)
     >>> device.open()
 
+
+The transport argument
```

Comment:
I would move this section and the table immediately after the list of the optional args.